src/Microsoft.DotNet.Darc/src/Darc/Options/GatherDropCommandLineOptions.cs

```diff
@@ -73,6 +74,8 @@ internal class GatherDropCommandLineOptions : CommandLineOptions
         [Option("latest-location", HelpText = "Download assets from their latest known location.")]
         public bool LatestLocation { get; set; }
 
+        [Option("sas-suffixes", Separator = ',', HelpText = "List of potential SAS suffixes that can be used if anonymous access to a blob uri fails.")]
```

Comment:
Maybe an example for how these should look? Not obvious from the text whether I need to pass the "?" from the SAS query string or not.